src/ORM/Hierarchy/Hierarchy.php

```diff
@@ -129,6 +134,28 @@ class Hierarchy extends Extension
      */
     private static $prepopulate_numchildren_cache = true;
 
+    /**
+     * The default method called on the Hierarchy class to get children
+     * This can be overriden on classes that use the Hierarchy extension, though it should only be
+     * defined on the base class that has the hierarchy extension applied to it. For instance:
+     * - MyBaseClass has the Hierarchy extension applied
+     * - MySubClass extends MyBaseClass
+     * Only define the default_children_method on MyBaseClass
+     */
+    private static string $default_children_method = 'getChildrenForTree';
```

Comment:
As per https://github.com/silverstripe/silverstripe-framework/pull/11722#discussion_r2087678833 this should be named something that makes it clear it's for tree generation, e.g. `tree_children_method`

Reply:
Updated

Reply:
Still not done? Did you forget to push?

Reply:
Opps sorry updated